sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -56,6 +56,10 @@ internal class RadarActivityLifecycleCallbacks : Application.ActivityLifecycleCa
         foreground = count > 0
 
         updatePermissionsDenied(activity)
+
+        Radar.logConversion("app_open", null) { status, event ->
```

Comment:
with the default value of `metadata` being null, we dont need to pass null here correct?

Reply:
this function signature didn't have a default value for `metadata`, i'll add that in